### DISCUSSION TITLE
Problema para acceder a netlify

### DISCUSSION THREAD

**Discussion by Plopez1015** [community/community#142176] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Other

### Body

Estimados buen dia , una consulta ustedes han sabido superar el error que figura en netlify, despues de la contraseña me pide un codigo de acceso en 2 pasos intento usar el de github pero tampoco me permite....

por favor me pueden ayudar...

**Comment by ebndev** [comment]
Hi @Plopez1015, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
